apiservers/portlayer/restapi/handlers/exec_handlers.go

```diff
@@ -75,10 +77,67 @@ func (handler *ExecHandlersImpl) Configure(api *operations.PortLayerAPI) {
 	}
 }
 
+func addContainerToScope(name string, ns *models.NetworkConfig) (*metadata.NetworkEndpoint, *network.Scope, error) {
+	if ns == nil {
+		return nil, nil, nil
+	}
+
+	var err error
+	var s *network.Scope
+	switch ns.NetworkName {
+	case "default":
+		s = GetNetworkContext().DefaultScope()
+
+	default:
+		var scopes []*network.Scope
+		scopes, err = GetNetworkContext().Scopes(&ns.NetworkName)
+		if err != nil || len(scopes) != 1 {
+			return nil, nil, err
+		}
+
+		s = scopes[0]
```

Comment:
I'm guessing the 0th element in this list has some special significance.  Comment please.